WordPress/Resources/release_notes.txt

```diff
@@ -1,10 +1,22 @@
+13.0
+-----
+Your phone has one time zone setting. Your website uses a different one. What are your site stats based on? Did that traffic spike happen this morning, or this afternoon? Answer: your stats are now based on the time zone settings of your website.
+
+Consistent color schemes with good contrast make an accessible, intuitive app, which makes happy users. Version 13.0 is more consistent; enjoy the happiness.
+
+The block editor keeps getting better and better. Not possible, you say? Pshaw, let us count the ways: it’s automatically enabled the first time you start a block-based post* and you can enable and disable it on a per-site basis, so every user gets the editor experience they want. Adding blocks from the post title shows an “add block here” indicator, to simplify content creation, and adding new blocks de-selects the post title so the focus is on the new block.
+
+* Unless you opted out in version 12.9! We’re not monsters.
```

Comment:
This should only contain the release notes for version 13.0. It should not have all the versions.

Reply:
Yep you're right! Thanks – fixed in bd1b81b0f1e5a2b5387c6cc233b4dd46eb1ab305

Reply:
Yep you're right! Thanks – fixed in bd1b81b0f1e5a2b5387c6cc233b4dd46eb1ab305